civitai-downloader-v2/tests/unit/test_streaming_search.py

```
@@ -0,0 +1,350 @@
+            ]
+            
+            # Test retry mechanism exists
+            stream = streaming_search.search_models_stream(search_params)
+            if hasattr(streaming_search, 'enable_retry'):
+                streaming_search.enable_retry(max_retries=1)
+                
+                models = []
+                try:
+                    async for model in stream:
+                        models.append(model)
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)